Mathlib/RingTheory/DedekindDomain/AdicValuation.lean

```diff
@@ -229,6 +230,10 @@ def intValuation : Valuation R ℤₘ₀ where
   map_add_le_max' := IntValuation.map_add_le_max' v
 #align is_dedekind_domain.height_one_spectrum.int_valuation IsDedekindDomain.HeightOneSpectrum.intValuation
 
+
+theorem intValuation_apply {R : Type _} [CommRing R] [IsDedekindDomain R] {r : R}
```

Comment:
```suggestion
theorem intValuation_apply {R : Type *} [CommRing R] [IsDedekindDomain R] {r : R}
```
I guess there already is an `R` in scope.

Reply:
Good catch.